lib/portage/process.py

```diff
@@ -704,6 +706,8 @@ def _exec(binary, mycommand, opt_name, fd_pipes,
 									noiselevel=-1)
 								os._exit(1)
 						if unshare_net:
+							# use 'localhost' to avoid hostname resolution problems
+							socket.sethostname('localhost')
```

Comment:
Maybe we should treat an exception as a warning here.

Reply:
And if it fails, we can include in the message that it's for FEATURES=network-sandbox (like the other `"Unable to unshare: %s (for FEATURES=\"%s\")\n"` message.